### PR TITLE
added lottie-react-native and in-between transition view

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -128,6 +128,7 @@ android {
 }
 
 dependencies {
+    compile project(':lottie-react-native')
```
Comment (lyw0149) [inline android/app/build.gradle:131]: is this good without exclude? just curious.

File: src/components/globalViews/userModeTransition.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            loop
+          />
+          <Text style={{
+            fontSize: 24,
+            fontWeight: '600',
+            color: 'black',
+            top: -24,
+            alignSelf: 'center'
+          }}>
+            {(this.props.isRunner) ? '러너로 변신중' : '오더로 변신중'}
```
Comment (lyw0149) [inline src/components/globalViews/userModeTransition.js:84]: let's talk about '변신중' later. good.

**Review by lyw0149 — COMMENTED** [review]
support android?

File: android/app/build.gradle (hunk 1 above)
Comment (ochanje210) [reply android/app/build.gradle:131]: it's been working without a problem.

File: src/components/globalViews/userModeTransition.js (hunk 2 above)
Comment (ochanje210) [reply src/components/globalViews/userModeTransition.js:84]: sure

**Review by lyw0149 — APPROVED** [review]

**Pull request merged**